File: instrumentation/src/main/scala/org/excavator/boot/instrumentation/application/MyAtm.scala
package org.excavator.boot.instrumentation.application

import org.slf4j.LoggerFactory

class MyAtm {
}

object MyAtm{
  val account = 10
  val logger = LoggerFactory.getLogger(classOf[MyAtm])
  def withdrawMoney(amount: Int) = {
    Thread.sleep(20001)
    logger.info(s"[Application] successful withdrawal of [{}] units ${amount}")
  }
}

File: instrumentation/src/main/java/org/excavator/boot/instrumentation/application/MyAtm.java
package org.excavator.boot.instrumentation;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.util.concurrent.TimeUnit;

public class MyAtm {
    private final static Logger logger = LoggerFactory.getLogger(MyAtm.class);

    public static void withdrawMoney(int amount){
        try {
            TimeUnit.SECONDS.sleep(1);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        logger.info("[Application] successful withdrawal of [{}] units", amount);
    }
}
